test.disabled/ejabberd_tests/tsung/mam_con.xml

```diff
@@ -58,7 +58,7 @@
       <!-- Returns a random message id of 24 hours age -->
       <![CDATA[
         fun() ->
-            Now = now_to_microseconds(now()),
+            Now = now_to_microseconds(os:timestamp()),
```

Comment:
They still can be useful for some user but I absolutely don't mind removing them. :)